books/clients/livelibru.py

```diff
@@ -0,0 +1,89 @@
+import requests
+import urllib
+import json
+import math
+
+REQUIRED_HEADERS = {
+    "Host": "www.livelib.ru",
+    "Accept-Language": "en-US;q=1.0, ru-US;q=0.9, be-US;q=0.8",
+    "Accept": "*/*",
+    "Connection": "keep-alive",
+    "Accept-Encoding": "gzip;q=1.0, compress;q=0.5",
+    "User-Agent": "LiveLib/3.24.5 (ru.livelib.LiveLib; build:3.24.5; iOS 16.5.1) Alamofire/3.24.5"
+}
+
+
+def _generate_session_guid():
+    init_session_response = requests.get(
+        url="https://www.livelib.ru/apiapp/v2.0/appsessions/me?andyll=and7mpp4ss&fields=guid",
+        headers=REQUIRED_HEADERS
+    )
+    if init_session_response.status_code > 299:
+        raise RuntimeError(f"Filed to start new livelib session, {init_session_response.text}, " +
+                           f"response status: {init_session_response.status_code}")
+    guid = json.loads(init_session_response.text).get("data").get("guid")
+    return guid
+
+
+def _get_another_page(query_text, session_token=None, start_from=1, result_count=100):
```

Comment:
You never pass result_count. Let's make 100 into a constant and hardcode use the constant, given that it's used in 2 different places.